### ISSUE TITLE
{Freeway-Bug} [Adaptive Cards> Product Video]: Voice Over does convey the incorrect name for the 'Unmute' button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#7588]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.3.1-beta.20220620.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 15.5
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.20220612.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.1" control and select it.
5. Navigate to 'Scenarios' control and activate it.
6. Navigate to 'ProductVideo.json' and double tap to activate it.
7. Navigate to 'Unmute' button and observe the issue.

**Actual Result:**
While navigating to the 'Unmute' button, screen reader is conveying the incorrect information as **'Mute button' dimmed**.

**Expected Result:**
Screen Reader should convey the correct information for the 'Unmute' button and when focus is landing on it then it should announce as **'Unmute button'**.

**Note:**
When user is changing the state from 'Mute to Unmute then it is announcing as mute for the unmute state.
Also it is conveying the incorrect information as 'Dimmed' for the 'Mute and Unmute' button. 

**User Experience:**
It will impact the screen reader user as user will not be able to know whether the control is mute or unmute.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.01.02

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/176428487-64023cea-e13b-4dd2-8c30-8370cd8f2cbd.MP4

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
@vagpt, this is Apple's default control and we don't have control over this player control unless Apple can address this issue, there won't be any fix.

This issue doesn't seem to deserve Sev2. A11y label for the button works. The button is a mute button that mutes the sound. therefore, announcing  mute button is correct.  It also announces the state of button when it's muted.
In summary, 
1. when unmuted, voice over announces mute button.
2. when muted, voice over announces mute muted button.
This seems reasonable.
What's annoying and not the best user experience is that it will say "dimmed", although it's annoying, a11y labels describes about the button perfectly, so it should be sev3.

I'm closing this issue because it's Apple's native control that we don't have control over.

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as per the above comment.
